hardware/scripts/gen_trace.py

```diff
@@ -718,7 +718,25 @@ def perf_metrics_to_csv(perf_metrics: list, filename: str):
         'stall_raw_lsu',
         'stall_raw_acc',
         'stall_lsu',
-        'stall_acc']
```

Comment:
I think those changes are already merged. You should rebase onto the current main.